cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1234,6 +1234,9 @@ declare namespace Deno {
      * Requires `allow-read` permission.
      */
     caFile?: string;
+
+    /** Request timeout in milliseconds. Sets deadline for total request + body read. */
+    timeoutMs?: number;
```

Comment:
Maybe just `timeout`.

Reply:
I guess `ms` is pretty standard in JS land. 👍